src/app/portlets/dot-templates/dot-template-create-edit/dot-template-advanced/dot-template-advanced.component.html

```diff
@@ -1,5 +1,7 @@
 <dot-portlet-base>
-    <dot-portlet-toolbar [actions]="actions$ | async"></dot-portlet-toolbar>
+    <dot-portlet-toolbar [actions]="actions$ | async">
+        <dot-global-message right></dot-global-message>
```

Comment:
Does this also shows in the design templates?

Reply:
yes already have it. because re-use the same component as layout in edit mode